modules/ols-about-running-openshift-lightspeed-in-disconnected-mode.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-running-openshift-lightspeed-in-disconnected-mode_{context}"]
+= About running {ols-long} in disconnected mode
+
+The {ols-long} OpenShift operator and the {ols-long} service can work in a disconnected environment. A disconnected environment is an environment that does not have full access to the internet. 
```

Comment:
`Openshift Lightspeed Openshift operator`?
IMO we don't need the second openshift, `{ols-long} operator `should be enough

Reply:
I completely missed this. Thanks!